CMakeLists.txt

```diff
@@ -591,6 +622,8 @@ elseif(BASEARCH_X86_FOUND)
     if(NOT ${ARCH} MATCHES "x86_64")
         add_feature_info(SSE2 1 "Support the SSE2 instruction set, using \"${SSE2FLAG}\"")
     endif()
+elseif(BASEARCH_PPC_FOUND)
```

Comment:
Don't forget this alphabetical too. Sorry for all these nit-picky things. It will make it easier to read in the end and more consistent.

Reply:
No problem. Changed.